engine/lib/phx/script/meta/Engine.lua

```diff
@@ -17,8 +17,9 @@ function Engine.Abort() end
 ---@return integer
 function Engine.GetBits() end
 
+---Return time passed since engine start.
 ---@return number
-function Engine:getTime() end
+function Engine:elapsedTime() end
```

Comment:
i would keep this as getTime tbh. With elapsed time i would expect "elapsed time since something i defined", we also have a different elapsed function for time stamps.

Reply:
reverted